src/services/OrderService.ts

```diff
@@ -290,6 +290,37 @@ const retryShippingLabel = async (shipmentIds: Array<string>): Promise<any> => {
   })
 }
 
+const fetchShipmentLabelError = async (shipmentIds: Array<string>): Promise<any> => {
+  let shipmentLabelError = [];
+  const params = {
+    "entityName": "ShipmentPackageRouteSeg",
+    "inputFields": {
+      "shipmentId": shipmentIds,
+      "gatewayMessage": null,
+      "gatewayMessage_op": "not-equal",
+      "gatewayStatus": "error",
+      "gatewayStatus_op": "equals"
+    }
+  }
+
+  try {
+    const resp = await api({
+      url: "performFind",
+      method: "get",
+      params
+    })
+
+    if (!hasError(resp)) {
+      shipmentLabelError = resp?.data.docs;
+    } else if (!resp?.data.error || (resp.data.error && resp.data.error !== "No record found")) {
```

Comment:
There is no need to check for error conditions as hasError method handles this cases.